easybuild/easyblocks/i/imkl.py

```diff
@@ -77,6 +77,34 @@ def prepare_step(self, *args, **kwargs):
             super(EB_imkl, self).prepare_step(*args, **kwargs)
         else:
             super(EB_imkl, self).prepare_step(*args, **kwargs)
+        if self.cfg['interfaces']:
```

Comment:
@bartoldeman One more minor style issues: add an empty line above (and maybe also a comment explaining what's going on in the block below)